elasticdl/python/worker/worker.py

```diff
@@ -32,9 +32,11 @@
     Tensor,
     emplace_tensor_pb_from_ndarray,
     serialize_tensor,
-    tensor_pb_to_ndarray,
 )
-from elasticdl.python.common.tensor_utils import deduplicate_indexed_slices
+from elasticdl.python.common.tensor_utils import (
+    deduplicate_indexed_slices,
```

Comment:
I didn't see the usage of deduplicate_indexed_slices in this file

Reply:
The `deduplicate_indexed_slices ` is used at 

https://github.com/sql-machine-learning/elasticdl/blob/0a6858da862f6d3f15f41b2c4693078972f848c4/elasticdl/python/worker/worker.py#L513-L519

Reply:
I'm curious why the import is added in the change, but we can't see the usage in the new added change.
If the two methods are already used in the worker.py, they should be already imported.